Libraries/LibGUI/Slider.cpp

```diff
@@ -159,10 +159,15 @@ void Slider::mouseup_event(MouseEvent& event)
 
 void Slider::mousewheel_event(MouseEvent& event)
 {
+    auto acceleration_modifier = m_step;
+
+    if(event.modifiers() == KeyModifier::Mod_Ctrl && knob_size_mode() == KnobSizeMode::Fixed)
```

Comment:
Please use `clang-format`

Reply:
Oops I should maybe set my IDE up to format on save